omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -126,8 +126,7 @@ Using conda (preferred)
 
 #. Create virtual environments named ``omeropy`` and ``omeroweb``::
 
-    $ conda create -n omeropy -c ome python=3.6 zeroc-ice36-python omero-py
-    $ conda create -n omeroweb -c ome python=3.6 zeroc-ice36-python omero-web
+    $ conda create -n omeropy -c conda-forge python=3.8 zeroc-ice omero-py
```

Comment:
The command does not really map the sentences above which talk about 2 virtual environments. Should this block also create an `omeroweb` environment installing `omero-py` from `conda-forge` and add an extra `pip install omero-web`command post activation below?

Reply:
I will review.  I also think we should remove that walkthrough since we are no longer testing it.